extensions/amp-accordion/1.0/accordion.js

```diff
@@ -37,17 +38,6 @@ const EMPTY_EXPANDED_MAP = {};
 
 const generateSectionId = sequentialIdGenerator();
 
-const CHILD_STYLE = {
```

Comment:
These particular styles are not overridable. As such, we can continue to inline them. If you still want them in the JSS - it's totally reasonable, but we need to add `!important`.

Reply:
I personally prefer using JSS to keep all styling together.  I think it improves readability so the developer can quickly look into the one JSS file to see all styling.  I have added `!important`.  If there's something else I should consider, don't hesitate to let me know :).  Thanks!